app/configuration/config_loader.py

```diff
@@ -0,0 +1,125 @@
+        """Loads the configuration file."""
+        existing_paths = []
+
+        if isinstance(self.paths, str):
+            if os.path.isfile(self.paths):
+                existing_paths.append(self.paths)
+        elif isinstance(self.paths, tuple):
+            for path in self.paths:
+                if os.path.isfile(path):
+                    existing_paths.append(path)
+
+        if len(existing_paths) > 1:
+            print(
+                term.blue_on_black(
+                    "One or more configuration files were found to exist. "
+                    f"Using the one found at {existing_paths[0]}."
+                )
+            )
+
+        if (
+            not existing_paths
+            # The path is not in a default location,
+            # this means that it is None from an unrecognized system or was manually specified
+            and self.paths not in Configuration.configuration_paths.values()
+        ):
+            print(
+                term.red_on_black(
+                    "No configuration file found at that location or "
+                    "you are using a system with an unknown default configuration file location"
+                )
+            )
+            return False
+
+        elif (
+            # The path is in a default location, a path was probably not specified,
+            # unless the user pointed to one in the default location
+            not existing_paths
+            and self.paths in Configuration.configuration_paths.values()
+        ):
+            term.red_on_black(("No default configuration file found on your system"))
+            return False
+
+        # Choose the first path which is loaded
+        self.path_loaded = existing_paths[0]
+        self._load_yaml(term)  # Set config
+        return self.config
+
+    def config_loader_screen(self, term: Terminal) -> Union[dict, bool]:
+        """Screen showing the erros and configuration loaded by the module."""
+        print(term.home + term.clear + term.move_y(0))
+        print(term.bold_black_on_green(term.center("Loading Configurations")))
+        print("\n")
+
+        config = self.load_config(term)
+        if not config:
+            self.exit(term)
+            return False
+
+        # Configuration validator
+        response = fix_config(config, term)
+        if not response:
+            self.exit(term)
+            return False
+
+        print("\n\n")
+        print(
+            term.bold_black_on_green(
+                term.center("Successfully loaded configuration...")
+            )
+        )
+        time.sleep(5)
+        return response
+
+    @staticmethod
+    def exit(term: Terminal) -> None:
+        """Show the invalid configuration exit screen."""
+        print("\n\n")
+        print(
+            term.bold_white_on_red(
+                term.center("Error loading the configuration. Exiting in 10 seconds...")
+            )
+        )
+        time.sleep(10)
```

Comment:
reduce to 5 seconds

Reply:
Alright will do.

Reply:
Fixed in 93e2d2986b3dad7129d6cab4fea7024c359d0aea.